src/styles/blocks/micro-logo.scss

```
@@ -0,0 +1,7 @@
+.micro-logo {
+  width: 23px;
+  height: 16px;
+  background-image: url(../images/Icon-Corona-virus.svg);
```

```suggestion
  background-image: url(../images/Icon-Corona.svg);
```
This code will be in your portfolio. Not every recruiter will appreciate this joke :)